shenyu-dist/shenyu-admin-dist/src/main/resources/bin/stop.sh

```diff
@@ -22,7 +22,7 @@ cd `dirname $0`
 cd ..
 DEPLOY_DIR=`pwd`
 
-PIDS=`ps -eo pid,cmd |grep -v tail|grep -v grep|grep "$DEPLOY_DIR"|awk '{print $1}'`
+PIDS=`ps ax |grep -v tail|grep -v grep|grep "$DEPLOY_DIR"|awk '{print $1}'`
```

Comment:
support linux？

Reply:
Yes, I already verified it in ubuntu and centos.

![image](https://user-images.githubusercontent.com/26451038/161406708-5eda69b2-3321-43ff-9c24-d18f9368afb3.png)